src/bootstrap/angular.bootstrap.js

```diff
@@ -37,14 +42,28 @@ module.exports = function(dependentModulesMap) {
     }]);
   }
 
-  function onChangeLiveEditAngular(event, targetCSSClassName, doc) {
-    var elem = doc.querySelector(targetCSSClassName);
-
-    angularCompile(elem, event.srcElement.value);
+  function onChangeLiveEditAngular(target, value) {
+    angularCompile(target, value);
   }
 
-  window.onChangeLiveEditAngular = onChangeLiveEditAngular;
-  // debounce(onChangeLiveEditAngular, 200);
+  // Create a new Code Mirror instance for each editor
+  const liveEditors = document.getElementsByClassName('live-editor');
```

Comment:
I'm concerned that if someone has a page which has an Angular AND React component on it, that CodeMirror will initialise the "live-editor" elements twice. 

Also, if most of this CodeMirror-init code is identical for each framework, do you think it could go into its own module, and be required() by each framework bootstrap file?